BenchmarkTests/Benchmarks/Sources/Benchmarks.swift

```diff
@@ -152,4 +153,29 @@ public enum Benchmarks {
 
         OpenTelemetry.registerMeterProvider(meterProvider: meterProvider)
     }
+
+    /// Configure and register a OpenTelemetry Tracer.
+    ///
+    /// - Parameter configuration: The Benchmark configuration.
+    public static func tracer(with configuration: Configuration) {
```

Comment:
nit/ This API name has no verb, making it impossible to understand what `Benchmarks.tracer(with:)` will do. I'd assume it returns a tracer, but in fact it registers one. Same applies to `Benchmark.metrics(with:)`.

I had to scroll to this implementation from `AppDelegate's` call in order to understand what this code does.

Reply:
Yes indeed, I've renamed them to `enableMetrics` and `enableTracer` 👍